### PR TITLE
Neo7337/master

### PR DESCRIPTION
**Pull request by neo7337**
Adding a new Endpoint **/v3/covid-19/vaccineStats**

### CODE REVIEW

**Comment by neo7337** [comment]
I guess you are right, lemme incorporate the changes to the same vaccine interface only

File: scrapers/covid-19/getVaccine.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ const axios = require('axios');
 const csv = require('csvtojson');
 const cheerio = require('cheerio');
 const logger = require('../../utils/logger');
+const _ = require('underscore');
```
Comment (ebwinters) [inline scrapers/covid-19/getVaccine.js:5]: Do you really need this library?

File: scrapers/covid-19/getVaccine.js [hunk 2]
```diff
@@ -51,6 +60,8 @@ const getVaccineData = async (keys, redis) => {
 		const parsedData = await csv().fromString(data);
 		redis.set(keys.vaccine, JSON.stringify({
 			source: 'https://www.raps.org/news-and-articles/news-articles/2020/3/covid-19-vaccine-tracker',
+			totalCandidates: _.size(parsedData).toString(),
```
Comment (ebwinters) [inline scrapers/covid-19/getVaccine.js:63]: Again, you don't need `_` to get size.

Comment (neo7337) [reply scrapers/covid-19/getVaccine.js]: updated the PR removed, underscore.js

**Pull request merged**